### PR TITLE
Call Taker CSS Cleanup

### PR DESCRIPTION
**Pull request by miles-grant-ibigroup**
**Description:**
The advanced options no longer floats. This makes it easier to see what's where.

**PR Checklist:**
- [x] Does the code follow accessibility standards (WCAG 2.1 AA Compliant)?
- [x] Are all languages supported (Internationalization/Localization)?
- [x] Are appropriate Typescript types implemented?

**Before:**
Screenshot 2023-06-07 at 1 08 59 PM 

**After:** 
Screenshot 2023-06-07 at 1 07 39 PM

### CODE REVIEW

File: lib/components/app/call-taker-panel.js [hunk 1]
```diff
@@ -147,12 +147,11 @@ class CallTakerPanel extends Component {
     const { expandAdvanced, planTripClicked } = this.state
     const advancedSearchStyle = {
       background: 'white',
+      boxShadow: 'none',
```
Comment (binh-dam-ibigroup) [inline lib/components/app/call-taker-panel.js:150]: Maybe move all of these styles to `.otp .advanced-search-options` in `calltaker.css`, instead, except for `display`. Also, `background`, `left`, and `right` can probably be deleted.

**Review by binh-dam-ibigroup — APPROVED** [review]
LGTM with one minor change.

File: lib/components/app/call-taker-panel.js [hunk 2]
```diff
@@ -147,12 +147,11 @@ class CallTakerPanel extends Component {
...
       display: expandAdvanced ? undefined : 'none',
```
Comment (binh-dam-ibigroup) [inline lib/components/app/call-taker-panel.js:151]: Actually, `expandAdvanced` into whether to render the thing at all.

Comment (miles-grant-ibigroup) [reply lib/components/app/call-taker-panel.js:151]: I think I tried this and it causes problems with the state because the forms disappear

**Review by daniel-heppner-ibigroup — APPROVED** [review]
nice small improvement! Thanks!

**Pull request merged**